hbase-server/src/main/java/org/apache/hadoop/hbase/executor/ExecutorService.java

```diff
@@ -182,12 +182,60 @@ public void delayedSubmit(EventHandler eh, long delay, TimeUnit unit) {
     return ret;
   }
 
+  /**
+   * Configuration wrapper for {@link Executor}.
+   */
+  public static class ExecutorConfig {
+    // Refer to ThreadPoolExecutor javadoc for details of these configuration.
+    // Argument validation and bound checks delegated to the underlying ThreadPoolExecutor
+    // implementation.
```

Comment:
@wchevreuil @virajjasani I lost the old comments after a force push. I think I updated the code to get rid of maxPoolSize which is not honored for unbounded taskQueue (thanks Wellington)